Snippets/Push/01-push-notifications.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  01-push-notifications.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+
+// snippet.import
+import PubNubSDK
+import Foundation
+
+// snippet.end
+
+// snippet.pubnub
+// Initializes a PubNub object with the configuration
+let pubnub = PubNub(
+  configuration: PubNubConfiguration(
+    publishKey: "demo",
+    subscribeKey: "demo",
+    userId: "myUniqueUserId"
+  )
+)
+
+// snippet.end
+
+// snippet.add-apns-devices-on-channels
+// Adding a device to a channel for APNs
+pubnub.addAPNSDevicesOnChannels(
+  ["channelSwift"],
+  device: Data([0x01, 0x02, 0x03, 0x04]), // Replace with actual device token
```

Comment:
btw, for such cases, when user need to replace something with own data, we can use this template syntax:
```
<#Data([0x01, 0x02, 0x03, 0x04])#>
```
And when user will copy&paste the code, placeholders will be shown like this and user will have to replace them: 
![image](https://github.com/user-attachments/assets/e75fe76f-c36f-405e-ac2e-bdbb81e87a9d)

Reply:
On second thought, such a file will not compile due to placeholders left in it. I will verify it and let you know

Reply:
@jguz-pubnub it won't compile. This is visual and compile-time notice for the user ;)